src/clangd-context.ts

```diff
@@ -67,7 +67,7 @@ export class ClangdContext implements vscode.Disposable {
       return;
 
     const clangd: vscodelc.Executable = {
-      command: clangdPath,
+      command: `"${clangdPath}"`,
```

Comment:
Could you add a comment please, something like:

```
// Quote the path. With `shell: true`, this is needed
// in case the path contains spaces.
```